### ISSUE TITLE
Any ReactJs version of this?

### ISSUE THREAD

**Issue by ucefkh** [internetarchive/bookreader#1222]
Hello guys, I was just wondering if there would be any ReactJs version of this?

thanks

**Comment by sharabhh** [comment]
I don't think so that this is an issue, can you please close it @ucefkh thank you

**Comment by ucefkh** [comment]
yes, this is a question so can i get an answer please?
